widget/embedded/src/hooks/useSwapInput.ts

```diff
@@ -59,8 +59,9 @@ export function useSwapInput(): UseSwapInput {
     affiliateRef,
     affiliateWallets,
     disabledLiquiditySources,
+    fetchStatus,
   } = useAppStore();
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
In the initial load, there is an instant jump when the URL params are not empty.

in https://widget.rango.exchange:
[Widget_Rango](https://github.com/rango-exchange/rango-client/assets/123297941/d473270e-095b-4998-b00b-9561ba6cce8f)

in local:
[Local](https://github.com/rango-exchange/rango-client/assets/123297941/b4396134-a8c5-421d-97f6-8a9f2a6aba4d)

my params were:
/?toBlockchain=STARKNET&toToken=USDT--0x68f5c6a61780768455de69077e07e89787839bf8166decfbf92b645209c0fb8&fromAmount=11111